cfep-05.md

```diff
@@ -0,0 +1,135 @@
+
+<table>
+<tr><td> Title </td><td> Providing pre-release builds on the conda-forge anaconda channel </td>
+<tr><td> Status </td><td> Proposed </td></tr>
+<tr><td> Author(s) </td><td> Eric Dill &lt;thedizzle@gmail.com&gt;</td></tr>
+<tr><td> Created </td><td> Sept 09, 2016</td></tr>
+<tr><td> Updated </td><td> Sept 09, 2016</td></tr>
+<tr><td> Discussion </td><td> NA </td></tr>
+<tr><td> Implementation </td><td> NA </td></tr>
+</table>
+
+## Abstract
+
+There are a number of recipe maintainers who have asked for guidance regarding
+how to make pre-release builds available with the CI infrastructure on
+conda-forge. This document outlines how those builds should be labeled on
+anaconda.org so that the more adventurous users can try out these pre-release
+packages.
+
+## Motivation
+
+Pypi allows package maintainers to upload "prerelease" builds which users can
+then install via
+
+```
+pip install PACKAGE_NAME --pre
+```
+
+The same semantics are slightly more cumbersome within the `conda` ecosystem but
+are still manageable.  The analogous command with `conda` is (for example):
+
+```
+conda install -c conda-forge/label/rc_matplotlib matplotlib
+```
+
+We, as the conda-forge community need to arrive at a consensus for what labels
+we will affix to packages so that there is a consistent user experience.
+
+## Backwards Compatibility
+
+There are only minimal concerns of backwards compatibility.  There are a few
+relevant labels that package maintainers have affixed to their pre-release
+packages. These are "pre" (1), "prerelease" (2), "rc" (2) and "test" (1).  The
+numbers in parentheses represent the number of projects which have released
+these pre-release packages.  With six total projects using non-main labels, I do
+not think we have a serious concern with backwards compatibility due to changes
+that come about via this proposal.
+
+## Specification
+
+Mark the pre-release package on anaconda.org as "dev" or "rc". The semantics of
```

Comment:
For the TLDR; readers the `Mark [...] as "dev" or "rc"` may sound like labelling as `dev`/`rc` (full name, not affix).